Worktrolly/worktrolly
# feat(appThemeFix): Fixed the theme button behaviour

### Functionality:
Earlier the theme button in the profile component was not behaving correctly.

### Solution:
Earlier appTheme was sent as 'undefined' using data binding from the profile component to the theme component
Added a ngIf condition to load the theme component only after appTheme is fetched using the subscribe method in profile-component, as a result of which the data binding is now working correctly and the button behaviour is fixed. 

### Risk level:

### How to test:
firebase emulators:start
ng serve
1. Change theme to dark. 2. Go out of profile page. 3. Come back to profile page. 4.Check the theme button
